manual/mapper/README.md

```diff
@@ -13,6 +13,7 @@ It is published as two artifacts:
       <groupId>com.datastax.oss</groupId>
       <artifactId>java-driver-mapper-processor</artifactId>
       <version>${driver.version}</version>
+      <scope>provided</scope>
```

Comment:
Shouldn't we remove this XML snippet instead? It doesn't showcase the preferred way of referencing the annotation processor.

Reply:
Yes, the goal on this page was just to show the coordinates (which is why I omitted the scope initially), but maybe using XML is confusing.

I'll rearrange it to move everything to the "configuring" page.